internal/statestore/backfill.go

```diff
@@ -134,3 +144,64 @@ func (rb *redisBackend) UpdateBackfill(ctx context.Context, backfill *pb.Backfil
 
 	return nil
 }
+
+// IndexBackfill adds the backfill to the index.
+func (rb *redisBackend) IndexBackfill(ctx context.Context, backfill *pb.Backfill) error {
+	redisConn, err := rb.redisPool.GetContext(ctx)
+	if err != nil {
+		return status.Errorf(codes.Unavailable, "IndexBackfill, id: %s, failed to connect to redis: %v", backfill.GetId(), err)
+	}
+	defer handleConnectionClose(&redisConn)
+
+	// TODO: review usage of version or stick with Generation
```

Comment:
Here (and other todo) can you add a task on your backlog for this?  I think generation is going to work here, so I'm worried nobody is going to look at this code and the todo will stick around...

Reply:
Otherwise, create an issue on github, saying these todos need to be cleaned up, and reference the issue in the todo, eg

// TODO(#1234): clean the gutters

Reply:
Yes, absolutely! I think there's a conversation needed to happen so about the Version/Generation issue, and as soon that clears, i'll delete the TODOs i've put

Reply:
#1294 Was created

Reply:
Remove the todo, and this can be merged.

Reply:
Go for merging @Laremere 👍